lib/gui/modules/analytics.js

```diff
@@ -53,11 +70,18 @@ analytics.config(($mixpanelProvider) => {
 
     /* eslint-enable camelcase */
 
-    electron: app.getVersion(),
+    electron: process.versions.electron,
     node: process.version,
     arch: process.arch,
-    version: packageJSON.version
+    version: packageJSON.version,
+    OSPlatform: os.platform(),
```

Comment:
It's just that `OS` is upper case, the same way `JSON` in `packageJSON` is. Can change it to lower case if you prefer.